tests/br_tikv_outage/run.sh

```diff
@@ -54,7 +54,8 @@ hint_backup_start=$TEST_DIR/hint_backup_start
 hint_get_backup_client=$TEST_DIR/hint_get_backup_client
 
 
-cases=${cases:-'outage outage-after-request outage-at-finegrained shutdown scale-out'}
+# cases=${cases:-'outage outage-after-request outage-at-finegrained shutdown scale-out'}
+cases=${cases:-'outage outage-after-request shutdown scale-out'}
```

Comment:
It's a bad idea to delete some case, we should file an issue to record that if we want to disable it temporarily. And we should add the issue link at above of the L57 to tracking it.

Reply:
Got it. I will close the pr.